osu.Game.Rulesets.Mania/ManiaInputManager.cs

```diff
@@ -13,6 +13,11 @@ public ManiaInputManager(RulesetInfo ruleset, int variant)
             : base(ruleset, variant, SimultaneousBindingMode.Unique)
         {
         }
+
+        public RulesetKeyBindingContainer GetKeyBindingContainer()
+        {
+            return (RulesetKeyBindingContainer)KeyBindingContainer;
+        }
```

Comment:
kind of defeats the purpose of `KeyBindingContainer` being protected in the first place if it's going to be re-exposed as public like this. especially so that other rulesets will presumably also need this to implement touch.